packages/web3torrent/src/library/paid-streaming-extension.ts

```diff
@@ -111,13 +117,36 @@ export abstract class PaidStreamingExtension implements Extension {
   onMessage(buffer: Buffer) {
     try {
       const jsonData = bencode.decode(buffer, undefined, undefined, 'utf8');
-      this.messageBus.emit(PaidStreamingExtensionEvents.NOTICE, jsonData);
+      this.messageHandler(jsonData);
     } catch (err) {
-      log('ERROR: decoding', err);
+      log('ERROR: onMessage decoding', err);
       return;
     }
   }
 
+  protected messageHandler({command, data}) {
+    switch (command) {
+      case PaidStreamingExtensionNotices.ACK:
+        return;
```

Comment:
Do you think we ought to use either `return` or `break` exclusively?

Reply:
Umm.. maybe, the idea was to avoid an unending cycle of ACKs between peers, and repeating code...  but yeah, it gets harder to read. I could change it in another PR later

Reply:
If there's a good reason for the difference, then that's fine and you could probably just add a comment (or not). It just popped out at me when I scanned over the code.